grpc/cpp-cubic/README.md

```diff
@@ -58,6 +58,20 @@ FetchContent_Populate(sdk_cubic)
 add_subdirectory(${sdk_cubic_SOURCE_DIR}/grpc/cpp-cubic ${sdk_cubic_BINARY_DIR})
 ```
 
+### Windows Build
+When building for Windows, we recommend running CMake in the Visual
+Studio [command line](https://docs.microsoft.com/en-us/visualstudio/ide/reference/command-prompt-powershell?view=vs-2019).
+The VS command line will automatically set several environment variables
+that will be useful to CMake in defining compilers and other options.
+You will also need to download and install the Netwide Assmbler from
+[here](https://www.nasm.us/). Make sure it is available to your `PATH`
+so that CMake can find it (or specify it manually in the CMake
+configuration).
+
+Also note that gRPC doesn't really support building shared libraries
+on Windows. If you try it, you will likely get build errors, similar to
+what is reported [here](https://github.com/grpc/grpc/issues/25311).
```

Comment:
So does that mean that the default cmake build on Windows will build a static library?  If so, we should say so to reassure them that they'll be OK as long as they don't try to do anything fancy.  Or if there are particular config options they need to pass or avoid passing, specify them.

Reply:
Yes. The default will build a static library. It probably wouldn't hurt to mention that they shouldn't add `-DBUILD_SHARED_LIBS=TRUE` to the cmake command. Or if they do, to make it explicitly off `-DBUILD_SHARED_LIBS=FALSE`.